examples/main/views/custom-interaction.js

```diff
@@ -18,11 +18,13 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-/* window */
 import React, {Component} from 'react';
 import PropTypes from 'prop-types';
-import MapGL, {autobind} from 'react-map-gl';
+import autobind from 'react-autobind';
+import {InteractiveMap} from 'react-map-gl';
+import DeckGL, {ArcLayer} from 'deck.gl';
```

Comment:
I use the markers component to achieve this in the `interaction` example. It doesn't require any external dependencies.